### PR TITLE
Feature/infinitam rosified

### PR DESCRIPTION
**Pull request by ffurrer**
This PR just moves the modified infinitam code we had in a separate repo to this one.

### CODE REVIEW

**Comment by ffurrer** [comment]
test this please

File: InfiniTAM/package.xml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  <buildtool_depend>catkin_simple</buildtool_depend>
+
+  <depend>cv_bridge</depend>
+  <depend>glog_catkin</depend>
+  <depend>roscpp</depend>
+  <depend>rospy</depend>
+  <depend>pcl_catkin</depend>
+  <depend>pcl_conversions</depend>
+  <depend>pcl_ros</depend>
+  <depend>librealsense</depend>
```
Comment (ntonci) [inline InfiniTAM/package.xml:21]: Maybe this is not needed here

File: InfiniTAM/cmake/FindRealSense.cmake [hunk 2]
```diff
@@ -1,10 +1,10 @@
-# - Find OpenNI
+# - Find RealSense
```
Comment (ntonci) [inline InfiniTAM/cmake/FindRealSense.cmake]: This file is not needed anymore I think

**Review by ntonci — APPROVED** [review]
Took a quick look, should be good since it is only a move... Lgtm

**Pull request merged**